tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -119,35 +150,2072 @@ def test_get_metrics_and_expectations(
                 },
             },
         },
+        Domain(domain_type="column", domain_kwargs={"column": "vendor_id",}): {
+            "$parameter.column_distinct_values.count": {
+                "value": [
+                    3,
+                    3,
+                    3,
+                    2,
+                    2,
+                    3,
+                    3,
```

Comment:
Nitpick - I think we should use the `fmt: off / fmt: on` directive that `black` provides for situations like this.